pubsublite/spark-connector/spark_streaming_from_pubsublite_example.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+
+def spark_streaming_from_pubsublite(
+    project_number: int, location: str, subscription_id: str
```

Comment:
We should be consistent on using `region` throughout instead of `location`. 
```suggestion
    project_number: int, region: str, subscription_id: str
```

Reply:
The expected Pub/Sub Lite location (cloud region + zone id) is different from Dataproc regions (cloud region) in this guide.  See README where I provide a link to Pub/Sub Lite locations. Also updated `CLOUD_REGION` to `DATARPOC_REGION` and `LOCATION` to `PUBSUBLITE_LOCATION`.

Reply:
Noted, LGTM.